src/registry_schemas/schemas/business.json

```diff
@@ -79,6 +72,38 @@
         "XS"
       ]
     },
+    "alternateName": {
+        "type": "object",
+        "properties": {
+          "entityType": {
+            "$ref": "#/definitions/legalType"
+          },
+          "identifier": {
+            "$ref": "#/definitions/identifier"
+          },
+          "nameRegisteredDate": {
+            "type": "string",
+            "format": "date-time",
+            "title": "The timestamp of the registered date.",
+            "default": "",
+            "examples": ["1970-01-01T00:00:00+00:00"]
+          },
+          "nameStartDate": {
+            "type": "string",
+            "format": "date",
+            "title": "The start date of the alternate name",
+            "default": "",
+            "examples": ["1970-01-01"]
+          },
+          "operatingName": {
+            "type": "string",
+            "title": "The Operating Name of the business.",
+            "default": "",
+            "examples": ["operating_name"],
+            "pattern": "^(.*)$"
+          }
+        }
+      },
```

Comment:
Had a discussion with Thor about making some tweaks to `alternateName` and to also update it so we can support what we have under the `aliases` table currently as well.

Here's an example of what the new alternateNames will look like:

```json
{
  "business": {
    "alternateNames": [
      {
        "name": "SOME NAME TRANSLATION 1",
        "nameStartDate": "2022-05-01",
        "nameType": "TRANSLATION"
      },
      {
        "name": "SOME NAME TRANSLATION 2",
        "nameStartDate": "2022-05-03",
        "nameType": "TRANSLATION"
      },
      {
        "entityType": "GP",
        "identifier": "FM1000019",
        "name": "FOX FOOD CHAIN",
        "nameRegisteredDate": "2021-04-28T00:00:00+00:00",
        "nameStartDate": "2022-04-29",
        "nameType": "DBA"
      },
      {
        "entityType": "SP",
        "identifier": "FM1111111",
        "name": "SWIFT FOX SNACKS",
        "nameRegisteredDate": "2023-05-10T00:00:00+00:00",
        "nameStartDate": "2023-05-10"
      }
    ],
    "identifier": "FM1000019",
    "legalName": "FOX2 DO2, FOX3 DO3",
    "legalType": "GP"
  }
}
```

So we'll need to have the following updates to the `alternateName` definition here:
* Add `nameType` enum which can have values of `TRANSLATION` and `DBA`
* Rename `operatingName` to `name`
* Add `nameEndDate`  Note: end dated names will not be returned for a business by default but we need this for scenarios where we will return historical info